### PR TITLE
[#5561]migrate tests to junit5

### PR DESCRIPTION
**Pull request by fspirola**

### CODE REVIEW

**Comment by fspirola** [comment]
#issue5561

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (fab-10) [inline .vscode/settings.json:1]: Please do not include IDE files in commits

File: ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/peers/EnodeURLImplTest.java [hunk 2]
```diff
@@ -500,7 +500,7 @@ public void toURI_WithHostnameShouldWorkWhenDnsEnabledAndUpdateEnabled()
         "enode://"
             + VALID_NODE_ID
             + "@"
-            + InetAddress.getLocalHost().getCanonicalHostName()
+            + InetAddress.getLocalHost().getHostName()
```
Comment (fab-10) [inline ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/peers/EnodeURLImplTest.java:503]: Is this related to Junit5 migration?

**Review by fab-10 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (fspirola) [reply .vscode/settings.json]: IDE files removed from commit. Include .vscode/ in gitignore

File: ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/peers/EnodeURLImplTest.java (hunk 2 above)
Comment (fspirola) [reply ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/peers/EnodeURLImplTest.java:503]: No, just a mistake.

**Review by macfarla — APPROVED** [review]
LGTM

**Comment by macfarla** [comment]
this unit test failure seems related to your changes - you should be able to reproduce locally

> Task :besu:test

RlpBlockImporterTest > blockImport() FAILED
    java.lang.RuntimeException at RlpBlockImporterTest.java:80
        Caused by: java.util.concurrent.CompletionException at CompletableFuture.java:315
            Caused by: java.lang.IllegalStateException at RlpBlockImporter.java:230

**Review by macfarla — APPROVED** [review]
